data.sql

```diff
@@ -1,5 +1,13 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (1, 'Agumon', '2020-02-03', 0, 1, 10.23);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (2, 'Gabumon', '2018-11-15', 2, 1, 8);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (3, 'Pikachu', '2021-01-07', 1, 0, 15.04);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (4, 'Devimon', '2017-05-12', 5, 1, 11);
```

Comment:
- [ ] As I said earlier about the data type `neutered`, it should be boolean - `true or false`. Please ensure you fix this